starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,75 @@
+body {
+  font-family: 'Open Sans',sans-serif;
+  margin: 0;
+  border: 0;
+  padding: 0;
+}
+h1 {
```

Comment:
Be careful with tag selectors.  You're changing all h1's in the project.  It works fine for this one, but with bigger projects, it will become a problem.